Tactics.lean

```diff
@@ -7,3 +7,6 @@ Author(s): Shilpi Goel
 -- Import modules here that should be built as part of the library.
 import «Tactics».Sym
 import «Tactics».StepThms
+import «Tactics».Rename
```

Comment:
I'm wondering if these tactics about names are related and therefore could be put in the same file.

Reply:
While I see the appeal, I prefer this approach, where it's stupidly easy to find out where a certain tactic is implemented.